dialogflow/cloud-client/src/test/java/com/example/dialogflow/DetectIntentKnowledgeTest.java

```diff
@@ -37,20 +35,16 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/**
- * Integration (system) tests for {DetectIntentKnowledge, KnowledgeManagement, DocumentManagement}.
- */
+
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
-public class KnowledgeBaseManagementIT {
+public class DetectIntentKnowledgeTest {
 
   private static String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
   private static String TEST_KNOWLEDGE_BASE_ID = "MTA4MzE0ODY5NTczMTQzNzU2ODA";
   private static String TEST_DOCUMENT_ID = "MTUwNjk0ODg1NTU4NzkzMDExMg";
```

Comment:
Should we be concerned about these constants and running testing jobs in parallel. (ie. j8 & j11)

Reply:
Not for this one. I used to have these tests dynamically create these resources, but the API has to do some ML training for an indeterminate time on the created resources which would create a flaky test occasionally. There are some more involved work-arounds to get around this, but would significantly complicate the tests.

So instead having this resource be static and used for "prediction" should handle any number of parallel requests and all of them should get an identical response and only a change to the Document we created should break this.